### ISSUE TITLE
test_templates error with no Objective-C compiler installed

### ISSUE THREAD

**Issue by floppym** [mesonbuild/meson#7203]
On Gentoo Linux, we see the following error when running unit tests.
```
======================================================================
ERROR: test_templates (__main__.AllPlatformTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "run_unittests.py", line 3381, in test_templates
    self._run(self.setup_command + ['--backend=ninja', 'builddir'],
  File "run_unittests.py", line 1549, in _run
    raise subprocess.CalledProcessError(p.returncode, command, output=p.stdout)
subprocess.CalledProcessError: Command '['/usr/lib/python-exec/python3.8/python', '/home/floppym/src/meson/meson.py', '--backend=ninja', '--backend=ninja', 'builddir']' returned non-zero exit status 1.
...
meson builddir
ninja -C builddir


cc: error: /tmp/tmpgk3o0o55/builddir/meson-private/sanitycheckobjc.m: Objective-C compiler not installed on this system
The Meson build system
Version: 0.54.2
Source dir: /tmp/tmpgk3o0o55
Build dir: /tmp/tmpgk3o0o55/builddir
Build type: native build
Project name: tmpgk3o0o55
Project version: 0.1

meson.build:1:0: ERROR: ObjC compiler ccache cc can not compile programs.
```

**Comment by eli-schwartz** [comment]
This should have been fixed by commit 3e89c30bae2265e694212a5909a553de0484dc14, please confirm.

**Comment by eli-schwartz** [comment]
(It has been confirmed fixed.)
